src/sos.pc.in

```diff
@@ -0,0 +1,18 @@
+## Sandia OpenSHMEM package metadata
+## Requires pkg-config 0.28 or later
+
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: sos
+Description: @PACKAGE_NAME@
+URL: @PACKAGE_URL@
+Version: @VERSION@
+Libs: -L${libdir} @WRAPPER_COMPILER_EXTRA_LDFLAGS@ -lsma @WRAPPER_COMPILER_EXTRA_LIBS@
```

Comment:
Interesting: no need for `LDFLAGS` here?  And `CFLAGS` below?

Reply:
Nope, pkg-config is a little underwhelming like that.